Sprint-1/destructuring/exercise-1/exercise.js

```diff
@@ -6,10 +6,18 @@ const personOne = {
 
 // Update the parameter to this function to make it work.
 // Don't change anything else.
-function introduceYourself(___________________________) {
+function introduceYourself(person) {
+  const {name,age,favouriteFood}=person
```

Comment:
This works, and does use object destructuring, but the exercise is asking you to _only_ change line 9, and you both changed line 9 _and_ added line 10. Can you work out how to do this _without_ introducing a parameter called `person`?

Reply:
sure thanks! done